tests/layer_tests/tensorflow_tests/test_tf_Conv3DBackprop.py

```diff
@@ -0,0 +1,84 @@
+                         /                                                         /
+            Placeholder-/                                  Placeholder->Transpose-/
+        """
+
+        import tensorflow as tf
+
+        if dilations is None:
+            dilations = [1, 1, 1, 1, 1] #default value regarding Documentation
+        else:
+            pytest.skip('Dilations != 1 isn\' supported on CPU by Tensorflow')
+
+        tf.compat.v1.reset_default_graph()
+
+        # Create the graph and model
+        with tf.compat.v1.Session() as sess:
+            tf_input = tf.constant(input_shape)
+            tf_filter = tf.compat.v1.placeholder(tf.float32, input_filter, "Input")
+            tf_backprop = tf.constant(np.ndarray(out_backprop, dtype=np.float32))
+
+            tf.raw_ops.Conv3DBackpropInputV2(input_sizes = tf_input, filter = tf_filter, out_backprop = tf_backprop, strides = input_strides, padding = input_padding)
+
+            tf.compat.v1.global_variables_initializer()
+            tf_net = sess.graph_def
+
+        ref_net = None
+
+        return tf_net, ref_net
+
+    test_data = [
+        dict(input_shape=[1, 15, 15, 15, 1], input_filter=[1, 1, 1, 1, 1], out_backprop=[1, 15, 15, 15, 1], input_strides=[1, 1, 1, 1, 1], input_padding='SAME', dilations=None), 
+        dict(input_shape=[1, 15, 15, 15, 2], input_filter=[1, 1, 1, 2, 2], out_backprop=[1, 15, 15, 15, 2], input_strides=[1, 1, 1, 1, 1], input_padding='SAME', dilations=None), 
+        dict(input_shape=[1, 16, 16, 16, 3], input_filter=[2, 2, 2, 3, 3], out_backprop=[1, 8, 8, 8, 3], input_strides=[1, 2, 2, 2, 1], input_padding='SAME', dilations=None), 
+        dict(input_shape=[1, 10, 10, 20, 3], input_filter=[2, 2, 2, 3, 3], out_backprop=[1, 5, 5, 10, 3], input_strides=[1, 2, 2, 2, 1], input_padding='SAME', dilations=None), 
+        dict(input_shape=[1, 15, 15, 15, 1], input_filter=[1, 1, 1, 1, 1], out_backprop=[1, 15, 15, 15, 1], input_strides=[1, 1, 1, 1, 1], input_padding='VALID', dilations=None), 
+        dict(input_shape=[1, 15, 15, 15, 2], input_filter=[1, 1, 1, 2, 2], out_backprop=[1, 15, 15, 15, 2], input_strides=[1, 1, 1, 1, 1], input_padding='VALID', dilations=None), 
+        dict(input_shape=[1, 16, 16, 16, 3], input_filter=[2, 2, 2, 3, 3], out_backprop=[1, 8, 8, 8, 3], input_strides=[1, 2, 2, 2, 1], input_padding='VALID', dilations=None), 
+        dict(input_shape=[1, 10, 10, 20, 3], input_filter=[2, 2, 2, 3, 3], out_backprop=[1, 5, 5, 10, 3], input_strides=[1, 2, 2, 2, 1], input_padding='VALID', dilations=None), 
+        pytest.param(
+        dict(input_shape=[1, 16, 16, 16, 3], input_filter=[3, 3, 3, 3, 3], out_backprop=[1, 8, 8, 8, 3], input_strides=[1,2,2,2,1], input_padding='SAME', dilations=None),
+            marks=pytest.mark.precommit_tf_fe),
+        pytest.param(
+        dict(input_shape=[1, 16, 16, 16, 3], input_filter=[3, 3, 3, 3, 3], out_backprop=[1, 7, 7, 7, 3], input_strides=[1,2,2,2,1], input_padding='VALID', dilations=None),
```

Comment:
please try to use different filter shapes in these two samples

Reply:
done